test/.cache/v/cache/lastfailed

```diff
@@ -0,0 +1,3 @@
+{
+  "test_game.py::test_answer": true
+}
```

Comment:
Haha you either need to add specific files (which I do) or check with `git status` that you added the right files (which you can always do to double check).
If you want to check the diff of your staged files you can always do `git diff --cached`